h2o-py/h2o/frame.py

```diff
@@ -417,6 +420,9 @@ def show(self, use_pandas=False, rows=10, cols=200):
         if self._ex is None:
             print("This H2OFrame has been removed.")
             return
+        if not self._has_content():
+            print("This H2OFrame is not initialized.")
```

Comment:
What is the semantic different between `This H2OFrame is not initialized` and `This H2OFrame is not initialized.` Does the first one mean that the Frame exists but wasn't loaded from the backend? Just trying to understand all aspects. Thanks!

Reply:
"not initialized" basically means both:
- the frame has no id (not attached to backend)
- the frame has no valid expression DAG (i.e. children = None)

this should be the case only for frames created with empty constructor and which are not assigned a valid expr later.

FYI, when manipulating frames locally, local copies are created with valid expr but no id: these are valid initialized frames.

If you think another word than "initialized" would be more meaningful for the end user, I'm a taker

Reply:
I'm thinking more about uniting these two cases. From user perspective, the frame is still empty and we might just confuse by another message type. What do you think @seb-h2o?

Reply:
It depends what you mean by uniting.

If you just mean, displaying the same "This H2OFrame is empty" message, it's reasonable, but less informative.

However technically, with our current way of handling frame states, these 2 frames H2OFrame() and H2OFrame([]) are fundamentally different: the first one represents a frame waiting for an expression to be assigned (and it's heavily used internally this way), whereas the second one is a proper frame that gets synced with backend.

Reply:
So I'm fine with uniting the messages only

Reply:
Thanks.

When I think about it more the `H2OFrame()` is not that common case for the user, so I guess it's fine to provide more fine-grained message. I would print something like "This frame is empty & not initializes" So it contains information about the emptiness and the fact it's not tight to the backend

Reply:
ok, changed